src/Api/Controllers/AnnotationController.cs

```diff
@@ -145,6 +145,38 @@ public IActionResult Post(int parentId, int childId)
             return BadRequest();
         }
 
+        // Post api/Annotation/Tags/Instance/:firstId/Relation/:secondId
+
+        /// <summary>
+        /// Add Relation between the tag instance represented by {firstId} and {secondId}.
+        /// </summary>
+        /// <param name="firstId">ID of the first tag instance of the relation</param>
+        /// <param name="secondId">ID of the second tag instance of the relation</param>
+        /// <param name="name" optional="true">Relation name</param>
+        /// <response code="200">relation added</response>
+        /// <response code="403">User not allowed to add a relation</response>
+        /// <response code="400">Request was misformed</response>
+        [HttpPost("Tags/Instance/{firstId}/Relation/{secondId}")]
+        [ProducesResponseType(typeof(void), 200)]
+        [ProducesResponseType(typeof(void), 403)]
+        [ProducesResponseType(typeof(void), 400)]
+        public IActionResult PostTagRelation(int firstId, int secondId, string name = "")
```

Comment:
When testing this with postman, there are no issues (i.e. the `name` is correctly set). So there seems to be an issue with swagger misinterpreting something.

Adding [FromBody] to the parameter declaration does not help, this causes the parameter to disappear from the swagger doc. Also, adding [FromRoute] to the firstId and secondId parameters - which I suppose would be the correct way of declaring the webservice endpoint - makes the whole call disappear from the swagger doc. Thoughts?